tests/src/mender-update/cli/cli_test.cpp

```diff
@@ -2364,7 +2558,6 @@ SupportsRollback
 ArtifactCommit_Enter_01
 ArtifactCommit
 ArtifactCommit_Leave_01
-ArtifactCommit_Error_01
```

Comment:
Can you explain me this behavior change? This will be different from the daemon mode AFAICS

Reply:
This was a bug in the standalone mode of the C++ client. Daemon mode [already does this](https://github.com/mendersoftware/mender/blob/4.0.4/src/mender-update/daemon/state_machine/state_machine.cpp#L199). Our docs also [confirm](https://docs.mender.io/3.7/artifact-creation/state-scripts#transitions-and-ordering) this behavior.